docker/setup-scripts/self-signed-certificates.sh

```diff
@@ -34,20 +34,20 @@ BASE_DIR="$(git rev-parse --show-toplevel)"
 . "${DOCKER_DIR}/sample/env.dev.sh"
 
 echo '2. create the self-signed CAs' | colored_cat g
-sh "${DEV_CA_DIR}/create-self-signed-ca.sh"
+bash "${DEV_CA_DIR}/create-self-signed-ca.sh"
```

Comment:
for the same reason as https://github.com/AthenZ/athenz/pull/2348/files#r1357701224